test/tests/functional/pbs_alps_release_tunables.py

```diff
@@ -151,10 +149,11 @@ def test_alps_release_jitter(self):
                 retry -= 1
                 # variable 'out' is a list of tuples and every second element
                 # in a tuple is the matched log message
-                time_prev = get_epoch(out[0][1])
+                time_prev = self.get_epoch(out[0][1])
                 for data in out[1:]:
-                    time_current = get_epoch(data[1])
-                    self.assertLessEqual(time_current - time_prev, max_delay,
+                    time_current = self.get_epoch(data[1])
+                    self.assertLessEqual(math.floor(time_current - time_prev),
```

Comment:
You shouldn't need math.floor here.  Just compare the high res times against the real max_delay time.  The logs should not be spread out more than max_delay.  However, it may be possible that if the system is a bit slow and the random number generated for jitter makes us equal to max_delay, then the logs can be logged a bit slower and thus be farther apart than max_delay.  In which case I think you can increase the alps_release_wait_time to workaround the slower system.